pkg/protocol/commitment_verifier.go

```diff
@@ -20,7 +20,7 @@ type CommitmentVerifier struct {
 }
 
 func NewCommitmentVerifier(mainEngine *engine.Engine, lastCommonCommitmentBeforeFork *model.Commitment) *CommitmentVerifier {
-	committeeAtForkingPoint := mainEngine.SybilProtection.SeatManager().Committee(lastCommonCommitmentBeforeFork.Slot()).Accounts().IDs()
+	committeeAtForkingPoint := lo.Return1(mainEngine.SybilProtection.SeatManager().CommitteeInSlot(lastCommonCommitmentBeforeFork.Slot())).Accounts().IDs()
```

Comment:
`CommitteeInSlot(...)` can return nil now, so this looks dangerous if you ignore the bool return variable. (This pattern is repeated everywhere where `Commitee(...)` was used before)

Reply:
We need to discuss what to do if the committee does not exists for some reason, because this should not happen, but if it does then it's pretty severe.